lms/static/coffee/spec/instructor_dashboard/student_admin_spec.coffee

```diff
@@ -0,0 +1,166 @@
+describe 'StudentAdmin', ->
```

Comment:
@andy-armstrong let me try to explain the issue here. Student admin feature for already written in coffee script I have added entrance exam section inside it. so source code of new feature resides in `lms/static/coffee/src`. Now I'm trying to write unit test for new feature and if I write unit test in js i have to put them inside `lms/static/js/spec` and RequireJS load path for js_test is not configured to load files from `lms/static/coffee/src` which makes sense.You can check it in `lms/static/js_test.yml` But as a result of this I cannot load my feature source file in unit test. . What you suggest in this situation?